packages/editor/src/client/panel/articleMetadataPanel.tsx

```diff
@@ -183,12 +188,30 @@ export function ArticleMetadataPanel({value, onClose, onChange}: ArticleMetadata
               </FormGroup>
               <FormGroup>
                 <ControlLabel>{t('articleEditor.panels.title')}</ControlLabel>
-                <FormControl
-                  value={title}
-                  onChange={title => onChange?.({...value, title, slug: slugify(title)})}
-                />
+                <FormControl value={title} onChange={title => onChange?.({...value, title})} />
                 <HelpBlock>{t('articleEditor.panels.titleHelpBlock')}</HelpBlock>
               </FormGroup>
+              <FormGroup>
+                <ControlLabel>{t('articleEditor.panels.slug')}</ControlLabel>
+                <InputGroup style={{width: '100%'}}>
+                  <FormControl
+                    value={slug}
+                    onChange={slug => onChange?.({...value, slug})}
```

Comment:
Should we use "slugify" here as well?

Reply:
@bishoymelek : It was a UX decision that I didn't: if we had `slugify` in here, the user wouldn't be able to make any spaces or capital letters while writing. I assumed that some users would get confused by this. So I decided that they can write like a normal sentence but as soon as they switch focus to another field (that's why I have the `onBlur`) the field gets automatically slugified. Do you think that makes sense?

Reply:
Absolutely